addOns/openapi/src/main/resources/org/zaproxy/zap/extension/openapi/resources/Messages.properties

```
@@ -66,16 +63,14 @@ openapi.swaggerconverter.definitionurl = Definition URL: {0}
 openapi.swaggerconverter.definitionurl.errorsyntax = Invalid definition URL:\n{0}\nRefer to log for more details.
 openapi.swaggerconverter.definitionurl.missingcomponents = The definition URL does not have the scheme or authority component:\n{0}
 openapi.swaggerconverter.nourls = Unable to obtain any server URL from the definition.
-openapi.swaggerconverter.parse.defn.exception = Failed to parse swagger defn {0}
+openapi.swaggerconverter.parse.defn.exception = Failed to parse Swagger definition: {0}
 openapi.swaggerconverter.serverurl = Server URL: {0}
 openapi.swaggerconverter.targeturl.errorsyntax = Invalid target URL:\n{0}\nRefer to the help for the supported syntax.
 openapi.swaggerconverter.targeturl.invalid = Failed to create/normalise the target URL:\n{0}
 openapi.swaggerconverter.targeturl.missingcomponents = The target URL does not have the scheme or authority component:\n{0}
 
-openapi.topmenu.import.importopenapi = Import an OpenAPI definition from the local file system
-openapi.topmenu.import.importopenapi.tooltip = The file must be a formal described OpenAPI definition.
-openapi.topmenu.import.importremoteopenapi = Import an OpenAPI definition from a URL
-openapi.topmenu.import.importremoteopenapi.tooltip = The contents must be a formal described OpenAPI definition.
+openapi.topmenu.import.importOpenapi = Import an OpenAPI definition
```

```suggestion
openapi.topmenu.import.importOpenapi = Import an OpenAPI Definition
```


The dialog title had it right